lib/created_message_queue_handler.ts

```diff
@@ -73,7 +73,7 @@ const notificationsCollectionUrl = documentDbUtils.getCollectionUri(
   "notifications"
 );
 
-const storageConnectionString = getRequiredStringEnv("AzureWebJobsStorage");
+const storageConnectionString = getRequiredStringEnv("QueueStorageConnection");
```

Comment:
No documentation to update?

Reply:
there are no references to the different blob storages actually

Reply:
I mean, in the setup instructions

Reply:
same there (no mention of the two storages as its handled by the programmatic tasks)

Reply:
can we merge this one ?